src/components/berths/index.tsx

```diff
@@ -44,10 +44,8 @@ export default ({ filtered, filteredNot, onClick, selected }: BerthProps) => (
         excluded={true}
         key={berth.identifier}
         berth={berth}
-        // TODO: fix this
-        // tslint:disable-next-line: jsx-no-lambda
-        onClick={() => onClick(berth.identifier)}
-        selected={selected.includes(berth.identifier)}
+        onClick={() => onClick(berth)}
+        selected={!!selected.find(selectedBerth => selectedBerth.identifier === berth.identifier)}
```

Comment:
Use var in comment above here so we dont re-evaluate again

Reply:
It needs to be reevaluated because `berth` is different for each case.